### DISCUSSION TITLE
Why normalization inside ASR model is performed after specaugm ?

### DISCUSSION THREAD

**Discussion by popcornell** [espnet/espnet#5263] [General]
See: https://github.com/espnet/espnet/blob/aa88f3a1cd7d1046bab005000f0337adaf9dd9f0/espnet2/asr/espnet_model.py#L385

in my understanding it should be performed before, at least it is done in original specaugm paper. 
There they replaced with zeros because the features are supposed to be zero-mean.

**Comment by sw005320** [comment]
I see. Make sense.
@kamo-naoyuki, can you answer it?
